### PR TITLE
Delete evaluation publish and remind queue #3044

### PR DESCRIPTION
**Pull request by ymoctavia**
for #3044

### CODE REVIEW

File: src/test/testng.xml [hunk 1]
```diff
@@ -33,7 +33,7 @@
             <class name="teammates.test.cases.logic.SubmissionsAdjustmentTest" />
             <class name="teammates.test.cases.logic.SubmissionsLogicTest" />
             <class name="teammates.test.cases.logic.SubmissionTaskQueueLogicTest" />
-            <class name="teammates.test.cases.logic.EvaluationsEmailTaskQueueTest" />
```
Comment (damithc) [inline src/test/testng.xml]: Remove the file as well

File: src/main/webapp/WEB-INF/queue.xml [hunk 2]
```diff
@@ -24,6 +24,18 @@
...
+  <name>admin-prepare-email-task-queue</name>  
+  <rate>1/s</rate>
+  <bucket-size>1</bucket-size>  
+</queue>
+<queue>  
+  <name>admin-send-email-queue</name>  
+  <rate>20/s</rate>
+  <bucket-size>40</bucket-size>  
+</queue>
+
```
Comment (damithc) [inline src/main/webapp/WEB-INF/queue.xml]: This shouldn't be here right?

File: src/main/java/teammates/common/util/Const.java [hunk 3]
```diff
@@ -44,8 +44,8 @@
         public static final String FEEDBACK_SUBMISSION_ADJUSTMENT_TASK_QUEUE = 
                                 "feedback-submission-adjust-queue";
         
-        public static final String EVAL_PUBLISH_EMAIL_TASK_QUEUE = "evaluation-publish-email-queue";
-        public static final String EVAL_REMIND_EMAIL_TASK_QUEUE = "evaluation-remind-email-queue";
+//        public static final String EVAL_PUBLISH_EMAIL_TASK_QUEUE = "evaluation-publish-email-queue";
+//        public static final String EVAL_REMIND_EMAIL_TASK_QUEUE = "evaluation-remind-email-queue";
```
Comment (damithc) [inline src/main/java/teammates/common/util/Const.java]: Can remove permannetly

**Comment by damithc** [comment]
Some changes requested.

**Comment by ymoctavia** [comment]
Professor @damithc , this is updated

**Comment by damithc** [comment]
Can merge after addressing all comments.

**Pull request merged**